modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteClientCdcStreamer.java

```diff
@@ -127,4 +127,9 @@ public KafkaToIgniteClientCdcStreamer(
 
         caches.forEach(name -> A.ensure(clusterCaches.contains(name), name + " not exists!"));
     }
+
+    /** {@inheritDoc} */
+    @Override protected String getIgniteInstanceName() {
+        return client.toString();
```

Comment:
I'm afraid toString is not implemented for thin clients. Perhaps we can use something like 'thin' here.

Reply:
Refactored the naming for CDC metrics registries. This code is removed